configure.ac

```diff
@@ -80,13 +74,50 @@ AC_PATH_PROG([XMLTO], [xmlto], [no])
     [exit 1]
 [fi]
 
-AC_PATH_PROG([PYTHON_CONFIG], [python-config], [no])
-[if test "$PYTHON_CONFIG" = "no"]
+AC_ARG_WITH(python2,
+AS_HELP_STRING([--with-python2],[build Python2 support (default is YES)]),
+ABRT_PARSE_WITH([python2]))
+
+[if test -z "$NO_PYTHON2"]
 [then]
-    [echo "The python-config program was not found in the search path. Please ensure"]
-    [echo "that it is installed and its directory is included in the search path."]
-    [echo "Then run configure again before attempting to build ABRT."]
-    [exit 1]
+    AM_CONDITIONAL(BUILD_PYTHON2, true)
+    AC_PATH_PROG([PYTHON], [python], [no])
+    [if test "$PYTHON" == "no"]
+    [then]
+        [echo "The python program was not found in the search path. Please ensure"]
+        [echo "that it is installed and its directory is included in the search path or"]
+        [echo "pass --without-python2 to ./configure."]
+        [echo "Then run configure again before attempting to build ABRT."]
+        [exit 1]
+    [fi]
+
+    AC_PATH_PROG([PYTHON_CONFIG], [python-config], [no])
+    [if test "$PYTHON_CONFIG" = "no"]
+    [then]
+        [echo "The python-config program was not found in the search path. Please ensure"]
+        [echo "that it is installed and its directory is included in the search path or"]
+        [echo "pass --without-python2 to ./configure."]
+        [echo "Then run configure again before attempting to build ABRT."]
+        [exit 1]
+    [fi]
+
+    PYTHON_CFLAGS=`python-config --cflags 2> /dev/null`
+    PYTHON_LIBS=`python-config --libs 2> /dev/null`
+
+    AC_SUBST([PYTHON2_PREFIX], ['${prefix}'])
+    AC_SUBST([PYTHON2_EXEC_PREFIX], ['${exec_prefix}'])
+
```

Comment:
These lines are placed twice in the document (line 209). Please remove the next ones.

Reply:
Good catch. I updated the commit.